morty.go

```diff
@@ -991,6 +1004,7 @@ func main() {
 	version := flag.Bool("version", false, "Show version")
 	requestTimeout := flag.Uint("timeout", 2, "Request timeout")
 	socks5 := flag.String("socks5", "", "SOCKS5 proxy")
+	siteReplace := flag.Bool("replace", false, "Enable experimental site redirection")
```

Comment:
Instead of a bool switch this could be used to set the path of the json file.

Reply:
Also, I'd remove the "experimental" word.

Reply:
If that's the way you want to go we can do that, but I feel like using that as a path to the json file lends itself to being enabled by default. If we set a default people will automatically use it, if we don't nobody will know about it. It's your decision though.

Reply:
It can be empty by default

Reply:
Don't you think that will cause more people not to use it? I really think that's adding complexity, it's not a well known feature, how will they know sites.json is the site replacement file